lib/Differentiator/VisitorBase.cpp

```diff
@@ -179,11 +179,12 @@ namespace clad {
     return new (m_Context) DeclStmt(DGR, noLoc, noLoc);
   }
 
-  DeclRefExpr* VisitorBase::BuildDeclRef(DeclaratorDecl* D) {
+  DeclRefExpr* VisitorBase::BuildDeclRef(DeclaratorDecl* D,
+                                         const CXXScopeSpec* SS) {
```

Comment:
```suggestion
                                         const CXXScopeSpec* SS/*=nullptr*/) {
```

That's something useful we have been doing in the codebase. I thought clang-tidy would suggest it..